packages/react/src/components/TooltipDefinition/TooltipDefinition.js

```diff
@@ -37,11 +42,30 @@ const TooltipDefinition = ({
     {
       [`${prefix}--tooltip--${direction}`]: direction,
       [`${prefix}--tooltip--align-${align}`]: align,
+      [`${prefix}--tooltip--hidden`]: !allowTooltipVisibility,
     }
   );
+  const handleFocus = () => setAllowTooltipVisibility(true);
+  const handleEscKeyDown = event => {
```

Comment:
Lets move this function definition into `useEffect()` callback - Otherwise I'm not sure if `addEventListener()` call and `removeEventListener()` call are done with the same function refs.